src/Core/Random/UniformSphereSampler.hpp

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include <Core/Random/SphereSampler.hpp>
+
+namespace Ra {
+namespace Core {
+namespace Random {
+
+class RA_CORE_API UniformSphereSampler : public SphereSampler<UniformSphereSampler>
+{
+  public:
+    UniformSphereSampler() {};
+    ~UniformSphereSampler() {};
+
+    static std::pair<Vector2, Scalar> getPointImplem( UniformGenerator* generator );
+    static std::pair<Vector3, Scalar> getDirImplem( UniformGenerator* generator );
```

Comment:
```suggestion
    static std::pair<Vector3, Scalar> getDir( UniformGenerator* generator );
```

Reply:
This getDirImplem method is here for the CRTP SphereSampler in order to have both static methods AND polymorphism.